sections/discussion/scalability-discussion.tex

```diff
@@ -0,0 +1,85 @@
+\section{Scalability in the system}
+\paragraph{Scalability principles}
+In \autoref{sec:scalability} we determined that there are different types of scalability, and that when designing for scalability certain principles should be followed.
+In this section we will examine and discuss which of these principles we followed throughout the creation of the system.
+
+\begin{itemize}
+    \item \textbf{Design two of everything}: When designing two of everything it is important to identify if there is any single points of failure.
+    In our case that would be the API and the database.
+    For our API we could start multilpe machines that each run an API so that if one fails the others will take over the workload.
```

Comment:
```suggestion
    For our API we could start multiple machines that each run an API so that if one fails the others will take over the workload.
```